interfaces/mount/ns.go

```diff
@@ -38,7 +39,7 @@ func mountNsPath(snapName string) string {
 func runNamespaceTool(toolName, snapName string) ([]byte, error) {
 	mntFile := mountNsPath(snapName)
 	if osutil.FileExists(mntFile) {
-		toolPath := filepath.Join(dirs.DistroLibExecDir, toolName)
+		toolPath := cmd.InternalToolPath(toolName)
```

Comment:
Yes, I was wondering where this all ought to live. Ideas welcome, I'll happily move this elsewhere